sdk/core/core-amqp/src/retry.ts

```diff
@@ -237,7 +244,7 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
           targetDelayInMs / 1000,
           config.operationType
         );
-        await delay(targetDelayInMs);
+        await delay(targetDelayInMs, config.abortSignal);
```

Comment:
Pass in the error message as below to the `delay` function
`The retry operation has been cancelled`